pythonic.el

```diff
@@ -77,9 +77,10 @@
 
 (defun pythonic-remote-port ()
   "Get port of the connection to the remote python interpreter."
-  (let ((hostname (tramp-file-name-host (tramp-dissect-file-name (pythonic-aliased-path default-directory)))))
-    (when (s-contains-p "#" hostname)
-      (string-to-number (replace-regexp-in-string "\\`.*#" "" hostname)))))
+  (let ((port (tramp-file-name-port (tramp-dissect-file-name (pythonic-aliased-path default-directory)))))
+    (if (stringp port)
```

Comment:
What else can it be?

Reply:
emacs 25 returns `tramp-file-name-port` as a number, and emacs 26 returns it as a string. This little piece makes them compatible. I probably should add a comment in the code, because it looks indeed not clear